site/installing.md

```diff
@@ -216,15 +216,15 @@ Weave Cloud hosts the Scope UI for you, provides secure access control for your
 
 Sign up for a [Weave Cloud account](https://cloud.weave.works/) and obtain a token. Replace `<token>` with your token by running this on the master node or on whatever machine that has `kubectl` configured to authenticate to your Kubernetes cluster:
 
-    kubectl apply --namespace kube-system -f "https://cloud.weave.works/k8s.yaml?service-token=<token>&k8s-version=$(kubectl version | base64 | tr -d '\n')"
+    kubectl apply --namespace kube-system -f "https://cloud.weave.works/k8s/scope.yaml?service-token=<token>&k8s-version=$(kubectl version | base64 | tr -d '\n')"
```

Comment:
Just an observation:
- you could have `-n kube-system` if you wanted something shorter;
- you can keep `--namespace kube-system` if you want something more explicit.

And likewise for the second URL change.